stable/database/files/nuosm

```diff
@@ -78,7 +82,7 @@ function die() {
 function log() {
   local message="$@"
   if [ -n "$message" ]; then
-    echo "$( date "+%Y-%m-%dT%H:%M:%S.%3N%z" ) $message" | tee -a $LOGFILE
+    echo "$( date "+%Y-%m-%dT%H:%M:%S.%3N%z" ) INFO  [:${HOSTNAME}] nuosm ${message}" | tee -a "$LOGFILE" "$NUODB_EXTERNAL_STARTUP_LOG"
```

Comment:
With message level set to INFO, is there a potential to miss an actual error if one is logged by nuosm?

For example:

```
2021-06-29T14:23:55.948+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] Archive restore will be performed for archiveId=2, source=20210510T172407, type=backupset, strip=1
2021-06-29T14:23:55.951+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] Error while performing restore for archiveId=2: Backupset 20210510T172407 cannot be found in /var/opt/nuodb/backup
```

Reply:
@bkelly-ndb You have a good point here. In `nuosm` we didn't have notion of logging levels, however, we have something like tracing.

I've added a small changes so that the default messages are printed with `INFO`, those from `die`function are `ERROR` and those from `trace` function are `DEBUG`.

For example:

```
2021-07-02T12:45:53.967+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm ===========================================
2021-07-02T12:45:53.977+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm logsize=10892; maxlog=5000000
2021-07-02T12:45:53.979+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm Directory /var/opt/nuodb/archive/nuodb/demo exists
2021-07-02T12:45:53.983+0000 DEBUG [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm retrieving legacy restore request from KV store
2021-07-02T12:45:55.751+0000 DEBUG [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm checking archive raft metadata
2021-07-02T12:45:55.755+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm archiveId=1; DB=demo; hostname=sm-database-nuodb-cluster0-demo-hotcopy-0
2021-07-02T12:45:57.476+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm path=/var/opt/nuodb/archive/nuodb/demo; atoms=68; catalogs=183
2021-07-02T12:45:57.480+0000 DEBUG [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm reading restore request for archiveId=1
2021-07-02T12:45:59.097+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm Archive with archiveId=1 has been requested for a restore
2021-07-02T12:45:59.143+0000 DEBUG [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm evaluating restore request data
2021-07-02T12:45:59.150+0000 INFO  [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm Archive restore will be performed for archiveId=1, source=dummy, type=backupset, strip=1
2021-07-02T12:45:59.155+0000 DEBUG [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm performing restore for archiveId=1
2021-07-02T12:45:59.161+0000 ERROR [:sm-database-nuodb-cluster0-demo-hotcopy-0] nuosm Error while performing restore for archiveId=1: Backupset dummy cannot be found in /var/opt/nuodb/backup
```